README.md

```diff
@@ -23,18 +23,30 @@ npm install
 ```
 
 ## Usage
+####Run by Command Line Arguments
```

Comment:
Could you follow the correct markdown hierarchy here? (Sub headers of a second-level header are always third-level)